tests/unit/style/parsing/border.rs

```diff
@@ -22,10 +22,14 @@ fn border_image_shorthand_should_parse_when_all_properties_specified() {
 
     assert_eq!(result.border_image_source.unwrap(),
                parse_longhand!(border_image_source, "linear-gradient(red, blue)"));
-    assert_eq!(result.border_image_slice.unwrap(), parse_longhand!(border_image_slice, "30 30% 45 fill"));
```

Comment:
Normally linter allows up to 120 characters per line. But this is better, I guess.